application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -59,6 +59,20 @@ analytics.mostViewedPages.parameter.label.startDate=Start time of the date inter
 rendering.macro.mostViewedPages.description=When visitors search on your website, they are looking for a particular page, content, product, or service. This report lists the pages that were clicked the most after an internal search.
 rendering.macro.mostViewedPages.name=Most viewed pages
 
+## Entry Pages
+analytics.entryPages.header.actions=Actions
+analytics.entryPages.header.bounces.hint=Number of visits that started and ended on this page.
+analytics.entryPages.header.bounces=Bounces
+analytics.entryPages.header.entrances.hint=Number of visits that started on this page.
+analytics.entryPages.header.entrances=Entrances
+analytics.entryPages.header.pageTitle=Title
+analytics.entryPages.header.visits.hint=The number of visits that included this page. If a page was viewed multiple times during one visit, it is only counted once.
+analytics.entryPages.header.visits=Unique visitors
+analytics.entryPages.parameter.label.endDate=End time of the date interval used for this macro
+analytics.entryPages.parameter.label.startDate=Start time of the date interval used for this macro
+rendering.macro.entryPages.description=This report contains information about the entry pages that were used during the specified period. An entry page is the first page that a user views during their visit.
+rendering.macro.entryPages.name=Entry Pages
```

Comment:
Where is this ``entryPages`` macro defined? I can only see a ``mostViewedPages`` macro. If the intention was to translate the ``mostViewedPages`` macro then you're also missing the translation for the macro parameters (``startDate`` and ``endDate``).